types/bootstrap-3-typeahead/index.d.ts

```diff
@@ -66,7 +66,7 @@ declare namespace Bootstrap3Typeahead {
         /**
          * Call back function to execute after selected an item
          */
-        afterSelect?: (item: string) => void;
+        afterSelect?: (item: any) => void;
```

Comment:
The type must be the same as the source, that is the string | object, not any.

Reply:
OK, should `callback: any` on [line 14](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/69c672aece3d1eca0ddbb8929e5fa049a6a6a2b6/types/bootstrap-3-typeahead/index.d.ts#L14) also be changed?

Reply:
Yes, the callback is the same type of source, but its return is itself (an object).